tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -325,6 +327,50 @@ def test_multiple_fsx(
     run_benchmarks(remote_command_executor, scheduler_commands)
 
 
+@pytest.mark.usefixtures("instance")
+def test_fsx_file_cache(
+    os,
+    region,
+    create_fsx_file_cache,
+    pcluster_config_reader,
+    s3_bucket_factory,
+    clusters_factory,
+    scheduler_commands_factory,
+    test_datadir,
+):
+    """
+    Test existing Fsx file cache
+
+    Check fsx_fc_id provided in config file can be mounted correctly
+    """
+    # Create s3 bucket and upload test_file
+    bucket_name = s3_bucket_factory()
+    bucket = boto3.resource("s3", region_name=region).Bucket(bucket_name)
+    bucket.upload_file(str(test_datadir / "s3_test_file"), "s3_test_file")
+    partition = utils.get_arn_partition(region)
+    num_existing_fsx_file_caches = (
```

Comment:
What about exiting the test case when the partition is not "aws"?
Currently, if the partition is not "aws" the test would proceed attempting to create the cluster, but it would not be required.

Reply:
Got it.